app/pages/privacy-policy.spec.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import PrivacyPolicy from './privacy-policy';
+
+describe('PrivacyPolicy', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = shallow(<PrivacyPolicy />);
+  });
+
+  it('renders without crashing', () => {
+    const privacyPolicyContainer = wrapper.find('div.content-container');
+    assert.equal(privacyPolicyContainer.length, 1);
+  });
+
+  describe('heading', () => {
+    it('renders the page title', () => {
+      const headingElement = wrapper
+        .find('div.content-container').children()
+        .first()
+        .props().component;
+      assert.equal(headingElement, 'h1');
+    });
+
+    it('renders page title content', () => {
+      const headingContent = wrapper
+        .find('div.content-container').children()
+        .first()
+        .props().content;
+      assert.equal(headingContent, 'privacy.title');
+    });
+  });
+
+  describe('user agreement section', () => {
+    let markdownElements;
+
+    beforeEach(() => {
+      markdownElements = wrapper
```

Comment:
This again is hard to figure out what exactly is being targeted. Try to simplify the selectors.